k8s.io/configmap-nginx.yaml

```diff
@@ -187,6 +187,14 @@ data:
           rewrite ^/(.*)$                  https://storage.googleapis.com/kubernetes-release/$1 redirect;
         }
       }
+      server {
+        server_name discuss.kubernetes.io discuss.k8s.io;
+        listen 80;
+        listen 443 ssl;
```

Comment:
In order for SSL to work in nginx without browsers complaining you'll need to specify where the cert files are

Reply:
Something like:
```
listen   443;

ssl    on;
ssl_certificate    /etc/ssl/your_domain_name.pem; (or bundle.crt)
ssl_certificate_key    /etc/ssl/your_domain_name.key;
```

Reply:
Ignore me. I didn't read further down the file. 😄 ssl is configured elsewhere similarly